packages/antd/test/Form.test.js

```diff
@@ -94,4 +94,19 @@ describe("single fields", () => {
       .toJSON();
     expect(tree).toMatchSnapshot();
   });
+  test("field with description", () => {
+    const schema = {
+      properties: {
+        test: {
+          type: "string",
+          title: "test",
+          description: "test description",
+        }
+      }
+    };
+    const tree = renderer
+      .create(<Form schema={schema} />)
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
```

Comment:
One more thing. Can you add another test that sets `ui:description` and ensures that the description is also rendered there?

Reply:
@epicfaace I added the test here, could you please check? thanks! https://github.com/rjsf-team/react-jsonschema-form/pull/2644